### ISSUE TITLE
layout is not good and words are going out of the board

### ISSUE THREAD

**Issue by Ritikabiswas** [iamparas0/TIC-TAC-TOE#333]
![Screenshot 2024-10-20 122859](https://github.com/user-attachments/assets/6d40766b-84e1-4164-b80d-d01dbb5e4ed1)

**Comment by RitikaMaha** [comment]
can i work on this @iamparas0

**Comment by RitikaMaha** [comment]
> @RitikaMaha I have done this and submitted the pull request once @iamparas0 gets online he will accept it

Yeah ik that but I would also love to do some changes from my side.. if it is OK @iamparas0

**Comment by Ritikabiswas** [comment]
@iamparas0 I have made pull request; kindly check
